src/components/LogOut.tsx

```diff
@@ -0,0 +1,13 @@
+type LogoutProps = {
+    text: string,
+    logout: () => void,
+}
+
+function LogOut({text, logout}: LogoutProps) {
```

Comment:
Other option that I prefer for typing components is the following:

```
const LogOut:React.FC<LogoutProps> = ({text, logout}) => {
return (
        <button onClick={logout}>{text}</button>
    )
}
```

It helps me to distinguish if it's a React component or just a function without seeing the return of the function, but is just code style, so up to you.

Reply:
I like this idea but I need some docs to review what it means this 😓 do you have any link there?